front/pages/api/w/[wId]/spaces/[spaceId]/members.ts

```diff
@@ -92,5 +92,5 @@ async function handler(
 }
 
 export default withSessionAuthenticationForWorkspace(
-  withResourceFetchingFromRoute(handler, "space")
+  withResourceFetchingFromRoute(handler, { space: { requireCanList: true } })
```

Comment:
Let's add another layer in the PATCH with canWrite || canAdministrate

Reply:
actually, only canAdministrate , as space.updatePermissions do not allow canWrite